### PR TITLE
Closes #1074: Update `operator_test` to account for `uint64`

### LINKED ISSUE #1074: Update `operator_test` to account for `uint64`
Update `operator_test` to account for `uint64`
Part of #1011

### PR DESCRIPTION
**Pull request by pierce314159**
Closes #1074: Update `operator_test` to account for `uint64`

The output of `testAllOperators` detailing which operations are supported by numpy and us for `uint64` (and others) can be seen below. It's collapsed cause it's kinda a lot

EDIT: Output has been updated with `unrecognized type` check

<details>
  <summary>Output of testAllOperators</summary>

```
Operators:  frozenset({'&', '>>>', '>', '+', '<=', '**', '/', '<<', '%', '*', '//', '>>', '!=', '==', '>=', '|', '<<<', '^', '-', '<'})
Dtypes:  dict_keys(['int64', 'uint64', 'float64', 'bool'])
pdarrays: 
int64 :  [0 1 2 3 4 5 6 7 8 9]
uint64 :  [0 1 2 3 4 5 6 7 8 9]
float64 :  [0.000000e+00 2.222222e-01 4.444444e-01 6.666667e-01 8.888889e-01 1.111111e+00 1.333333e+00 1.555556e+00 1.777778e+00 2.000000e+00]
bool :  [True False True False True False True False True False]
ndarrays: 
int64 :  [0 1 2 3 4 5 6 7 8 9]
uint64 :  [0 1 2 3 4 5 6 7 8 9]
float64 :  [0.         0.22222222 0.44444444 0.66666667 0.88888889 1.11111111
 1.33333333 1.55555556 1.77777778 2.        ]
bool :  [ True False  True False  True False  True False  True False]
scalars: 
int64 :  5
uint64 :  5
float64 :  3.14159
bool :  True
# ops not implemented by numpy or arkouda: 103
int64(array) & uint64(array)
int64(array) << uint64(array)
int64(array) >> uint64(array)
int64(array) | uint64(array)
int64(array) ^ uint64(array)
int64(array) & float64(scalar)
int64(scalar) & float64(array)
int64(array) << float64(scalar)
int64(scalar) << float64(array)
int64(array) >> float64(scalar)
int64(scalar) >> float64(array)
int64(array) | float64(scalar)
int64(scalar) | float64(array)
int64(array) ^ float64(scalar)
int64(scalar) ^ float64(array)
uint64(array) & int64(array)
uint64(array) << int64(array)
uint64(array) >> int64(array)
uint64(array) | int64(array)
uint64(array) ^ int64(array)
uint64(array) & float64(array)
uint64(array) & float64(scalar)
uint64(scalar) & float64(array)
uint64(array) << float64(array)
uint64(array) << float64(scalar)
uint64(scalar) << float64(array)
uint64(array) >> float64(array)
uint64(array) >> float64(scalar)
uint64(scalar) >> float64(array)
uint64(array) | float64(array)
uint64(array) | float64(scalar)
uint64(scalar) | float64(array)
uint64(array) ^ float64(array)
uint64(array) ^ float64(scalar)
uint64(scalar) ^ float64(array)
float64(array) & int64(scalar)
float64(scalar) & int64(array)
float64(array) << int64(scalar)
float64(scalar) << int64(array)
float64(array) >> int64(scalar)
float64(scalar) >> int64(array)
float64(array) | int64(scalar)
float64(scalar) | int64(array)
float64(array) ^ int64(scalar)
float64(scalar) ^ int64(array)
float64(array) & uint64(array)
float64(array) & uint64(scalar)
float64(scalar) & uint64(array)
float64(array) << uint64(array)
float64(array) << uint64(scalar)
float64(scalar) << uint64(array)
float64(array) >> uint64(array)
float64(array) >> uint64(scalar)
float64(scalar) >> uint64(array)
float64(array) | uint64(array)
float64(array) | uint64(scalar)
float64(scalar) | uint64(array)
float64(array) ^ uint64(array)
float64(array) ^ uint64(scalar)
float64(scalar) ^ uint64(array)
float64(array) & float64(scalar)
float64(scalar) & float64(array)
float64(array) << float64(scalar)
float64(scalar) << float64(array)
float64(array) >> float64(scalar)
float64(scalar) >> float64(array)
float64(array) | float64(scalar)
float64(scalar) | float64(array)
float64(array) ^ float64(scalar)
float64(scalar) ^ float64(array)
float64(array) & bool(array)
float64(array) & bool(scalar)
float64(scalar) & bool(array)
float64(array) << bool(array)
float64(array) << bool(scalar)
float64(scalar) << bool(array)
float64(array) >> bool(array)
float64(array) >> bool(scalar)
float64(scalar) >> bool(array)
float64(array) | bool(array)
float64(array) | bool(scalar)
float64(scalar) | bool(array)
float64(array) ^ bool(array)
float64(array) ^ bool(scalar)
float64(scalar) ^ bool(array)
bool(array) & float64(array)
bool(array) & float64(scalar)
bool(scalar) & float64(array)
bool(array) << float64(array)
bool(array) << float64(scalar)
bool(scalar) << float64(array)
bool(array) >> float64(array)
bool(array) >> float64(scalar)
bool(scalar) >> float64(array)
bool(array) | float64(array)
bool(array) | float64(scalar)
bool(scalar) | float64(array)
bool(array) ^ float64(array)
bool(array) ^ float64(scalar)
bool(scalar) ^ float64(array)
bool(array) - bool(array)
bool(array) - bool(scalar)
bool(scalar) - bool(array)
# ops implemented by numpy but not arkouda: 516
int64(array) & uint64(scalar)
int64(scalar) & uint64(array)
int64(array) > uint64(array)
int64(array) > uint64(scalar)
int64(scalar) > uint64(array)
int64(array) + uint64(array)
int64(array) + uint64(scalar)
int64(scalar) + uint64(array)
int64(array) <= uint64(array)
int64(array) <= uint64(scalar)
int64(scalar) <= uint64(array)
int64(array) ** uint64(array)
int64(array) ** uint64(scalar)
int64(scalar) ** uint64(array)
int64(array) / uint64(array)
int64(array) / uint64(scalar)
int64(scalar) / uint64(array)
int64(array) << uint64(scalar)
int64(scalar) << uint64(array)
int64(array) % uint64(array)
int64(array) % uint64(scalar)
int64(scalar) % uint64(array)
int64(array) * uint64(array)
int64(array) * uint64(scalar)
int64(scalar) * uint64(array)
int64(array) // uint64(array)
int64(array) // uint64(scalar)
int64(scalar) // uint64(array)
int64(array) >> uint64(scalar)
int64(scalar) >> uint64(array)
int64(array) != uint64(array)
int64(array) != uint64(scalar)
int64(scalar) != uint64(array)
int64(array) == uint64(array)
int64(array) == uint64(scalar)
int64(scalar) == uint64(array)
int64(array) >= uint64(array)
int64(array) >= uint64(scalar)
int64(scalar) >= uint64(array)
int64(array) | uint64(scalar)
int64(scalar) | uint64(array)
int64(array) ^ uint64(scalar)
int64(scalar) ^ uint64(array)
int64(array) - uint64(array)
int64(array) - uint64(scalar)
int64(scalar) - uint64(array)
int64(array) < uint64(array)
int64(array) < uint64(scalar)
int64(scalar) < uint64(array)
int64(array) % float64(scalar)
int64(scalar) % float64(array)
int64(array) & bool(array)
int64(array) & bool(scalar)
int64(scalar) & bool(array)
int64(array) > bool(array)
int64(array) > bool(scalar)
int64(scalar) > bool(array)
int64(array) <= bool(array)
int64(array) <= bool(scalar)
int64(scalar) <= bool(array)
int64(array) ** bool(array)
int64(array) ** bool(scalar)
int64(scalar) ** bool(array)
int64(array) / bool(array)
int64(array) / bool(scalar)
int64(scalar) / bool(array)
int64(array) << bool(array)
int64(array) << bool(scalar)
int64(scalar) << bool(array)
int64(array) % bool(array)
int64(array) % bool(scalar)
int64(scalar) % bool(array)
int64(array) // bool(array)
int64(array) // bool(scalar)
int64(scalar) // bool(array)
int64(array) >> bool(array)
int64(array) >> bool(scalar)
int64(scalar) >> bool(array)
int64(array) != bool(array)
int64(array) != bool(scalar)
int64(scalar) != bool(array)
int64(array) == bool(array)
int64(array) == bool(scalar)
int64(scalar) == bool(array)
int64(array) >= bool(array)
int64(array) >= bool(scalar)
int64(scalar) >= bool(array)
int64(array) | bool(array)
int64(array) | bool(scalar)
int64(scalar) | bool(array)
int64(array) ^ bool(array)
int64(array) ^ bool(scalar)
int64(scalar) ^ bool(array)
int64(array) < bool(array)
int64(array) < bool(scalar)
int64(scalar) < bool(array)
uint64(array) & int64(scalar)
uint64(scalar) & int64(array)
uint64(array) > int64(array)
uint64(array) > int64(scalar)
uint64(scalar) > int64(array)
uint64(array) + int64(array)
uint64(array) + int64(scalar)
uint64(scalar) + int64(array)
uint64(array) <= int64(array)
uint64(array) <= int64(scalar)
uint64(scalar) <= int64(array)
uint64(array) ** int64(array)
uint64(array) ** int64(scalar)
uint64(scalar) ** int64(array)
uint64(array) / int64(array)
uint64(array) / int64(scalar)
uint64(scalar) / int64(array)
uint64(array) << int64(scalar)
uint64(scalar) << int64(array)
uint64(array) % int64(array)
uint64(array) % int64(scalar)
uint64(scalar) % int64(array)
uint64(array) * int64(array)
uint64(array) * int64(scalar)
uint64(scalar) * int64(array)
uint64(array) // int64(array)
uint64(array) // int64(scalar)
uint64(scalar) // int64(array)
uint64(array) >> int64(scalar)
uint64(scalar) >> int64(array)
uint64(array) != int64(array)
uint64(array) != int64(scalar)
uint64(scalar) != int64(array)
uint64(array) == int64(array)
uint64(array) == int64(scalar)
uint64(scalar) == int64(array)
uint64(array) >= int64(array)
uint64(array) >= int64(scalar)
uint64(scalar) >= int64(array)
uint64(array) | int64(scalar)
uint64(scalar) | int64(array)
uint64(array) ^ int64(scalar)
uint64(scalar) ^ int64(array)
uint64(array) - int64(array)
uint64(array) - int64(scalar)
uint64(scalar) - int64(array)
uint64(array) < int64(array)
uint64(array) < int64(scalar)
uint64(scalar) < int64(array)
uint64(array) & uint64(scalar)
uint64(scalar) & uint64(array)
uint64(array) > uint64(scalar)
uint64(scalar) > uint64(array)
uint64(array) + uint64(scalar)
uint64(scalar) + uint64(array)
uint64(array) <= uint64(scalar)
uint64(scalar) <= uint64(array)
uint64(array) ** uint64(scalar)
uint64(scalar) ** uint64(array)
uint64(array) / uint64(scalar)
uint64(scalar) / uint64(array)
uint64(array) << uint64(scalar)
uint64(scalar) << uint64(array)
uint64(array) % uint64(scalar)
uint64(scalar) % uint64(array)
uint64(array) * uint64(scalar)
uint64(scalar) * uint64(array)
uint64(array) // uint64(scalar)
uint64(scalar) // uint64(array)
uint64(array) >> uint64(scalar)
uint64(scalar) >> uint64(array)
uint64(array) != uint64(scalar)
uint64(scalar) != uint64(array)
uint64(array) == uint64(scalar)
uint64(scalar) == uint64(array)
uint64(array) >= uint64(scalar)
uint64(scalar) >= uint64(array)
uint64(array) | uint64(scalar)
uint64(scalar) | uint64(array)
uint64(array) ^ uint64(scalar)
uint64(scalar) ^ uint64(array)
uint64(array) - uint64(scalar)
uint64(scalar) - uint64(array)
uint64(array) < uint64(scalar)
uint64(scalar) < uint64(array)
uint64(array) > float64(array)
uint64(array) > float64(scalar)
uint64(scalar) > float64(array)
uint64(array) + float64(array)
uint64(array) + float64(scalar)
uint64(scalar) + float64(array)
uint64(array) <= float64(array)
uint64(array) <= float64(scalar)
uint64(scalar) <= float64(array)
uint64(array) ** float64(array)
uint64(array) ** float64(scalar)
uint64(scalar) ** float64(array)
uint64(array) / float64(array)
uint64(array) / float64(scalar)
uint64(scalar) / float64(array)
uint64(array) % float64(array)
uint64(array) % float64(scalar)
uint64(scalar) % float64(array)
uint64(array) * float64(array)
uint64(array) * float64(scalar)
uint64(scalar) * float64(array)
uint64(array) // float64(array)
uint64(array) // float64(scalar)
uint64(scalar) // float64(array)
uint64(array) != float64(array)
uint64(array) != float64(scalar)
uint64(scalar) != float64(array)
uint64(array) == float64(array)
uint64(array) == float64(scalar)
uint64(scalar) == float64(array)
uint64(array) >= float64(array)
uint64(array) >= float64(scalar)
uint64(scalar) >= float64(array)
uint64(array) - float64(array)
uint64(array) - float64(scalar)
uint64(scalar) - float64(array)
uint64(array) < float64(array)
uint64(array) < float64(scalar)
uint64(scalar) < float64(array)
uint64(array) & bool(array)
uint64(array) & bool(scalar)
uint64(scalar) & bool(array)
uint64(array) > bool(array)
uint64(array) > bool(scalar)
uint64(scalar) > bool(array)
uint64(array) + bool(array)
uint64(array) + bool(scalar)
uint64(scalar) + bool(array)
uint64(array) <= bool(array)
uint64(array) <= bool(scalar)
uint64(scalar) <= bool(array)
uint64(array) ** bool(array)
uint64(array) ** bool(scalar)
uint64(scalar) ** bool(array)
uint64(array) / bool(array)
uint64(array) / bool(scalar)
uint64(scalar) / bool(array)
uint64(array) << bool(array)
uint64(array) << bool(scalar)
uint64(scalar) << bool(array)
uint64(array) % bool(array)
uint64(array) % bool(scalar)
uint64(scalar) % bool(array)
uint64(array) * bool(array)
uint64(array) * bool(scalar)
uint64(scalar) * bool(array)
uint64(array) // bool(array)
uint64(array) // bool(scalar)
uint64(scalar) // bool(array)
uint64(array) >> bool(array)
uint64(array) >> bool(scalar)
uint64(scalar) >> bool(array)
uint64(array) != bool(array)
uint64(array) != bool(scalar)
uint64(scalar) != bool(array)
uint64(array) == bool(array)
uint64(array) == bool(scalar)
uint64(scalar) == bool(array)
uint64(array) >= bool(array)
uint64(array) >= bool(scalar)
uint64(scalar) >= bool(array)
uint64(array) | bool(array)
uint64(array) | bool(scalar)
uint64(scalar) | bool(array)
uint64(array) ^ bool(array)
uint64(array) ^ bool(scalar)
uint64(scalar) ^ bool(array)
uint64(array) - bool(array)
uint64(array) - bool(scalar)
uint64(scalar) - bool(array)
uint64(array) < bool(array)
uint64(array) < bool(scalar)
uint64(scalar) < bool(array)
float64(array) % int64(scalar)
float64(scalar) % int64(array)
float64(array) > uint64(array)
float64(array) > uint64(scalar)
float64(scalar) > uint64(array)
float64(array) + uint64(array)
float64(array) + uint64(scalar)
float64(scalar) + uint64(array)
float64(array) <= uint64(array)
float64(array) <= uint64(scalar)
float64(scalar) <= uint64(array)
float64(array) ** uint64(array)
float64(array) ** uint64(scalar)
float64(scalar) ** uint64(array)
float64(array) / uint64(array)
float64(array) / uint64(scalar)
float64(scalar) / uint64(array)
float64(array) % uint64(array)
float64(array) % uint64(scalar)
float64(scalar) % uint64(array)
float64(array) * uint64(array)
float64(array) * uint64(scalar)
float64(scalar) * uint64(array)
float64(array) // uint64(array)
float64(array) // uint64(scalar)
float64(scalar) // uint64(array)
float64(array) != uint64(array)
float64(array) != uint64(scalar)
float64(scalar) != uint64(array)
float64(array) == uint64(array)
float64(array) == uint64(scalar)
float64(scalar) == uint64(array)
float64(array) >= uint64(array)
float64(array) >= uint64(scalar)
float64(scalar) >= uint64(array)
float64(array) - uint64(array)
float64(array) - uint64(scalar)
float64(scalar) - uint64(array)
float64(array) < uint64(array)
float64(array) < uint64(scalar)
float64(scalar) < uint64(array)
float64(array) % float64(scalar)
float64(scalar) % float64(array)
float64(array) > bool(array)
float64(array) > bool(scalar)
float64(scalar) > bool(array)
float64(array) <= bool(array)
float64(array) <= bool(scalar)
float64(scalar) <= bool(array)
float64(array) ** bool(array)
float64(array) ** bool(scalar)
float64(scalar) ** bool(array)
float64(array) / bool(array)
float64(array) / bool(scalar)
float64(scalar) / bool(array)
float64(array) % bool(array)
float64(array) % bool(scalar)
float64(scalar) % bool(array)
float64(array) // bool(array)
float64(array) // bool(scalar)
float64(scalar) // bool(array)
float64(array) != bool(array)
float64(array) != bool(scalar)
float64(scalar) != bool(array)
float64(array) == bool(array)
float64(array) == bool(scalar)
float64(scalar) == bool(array)
float64(array) >= bool(array)
float64(array) >= bool(scalar)
float64(scalar) >= bool(array)
float64(array) < bool(array)
float64(array) < bool(scalar)
float64(scalar) < bool(array)
bool(array) & int64(array)
bool(array) & int64(scalar)
bool(scalar) & int64(array)
bool(array) > int64(array)
bool(array) > int64(scalar)
bool(scalar) > int64(array)
bool(array) <= int64(array)
bool(array) <= int64(scalar)
bool(scalar) <= int64(array)
bool(array) ** int64(array)
bool(array) ** int64(scalar)
bool(scalar) ** int64(array)
bool(array) / int64(array)
bool(array) / int64(scalar)
bool(scalar) / int64(array)
bool(array) << int64(array)
bool(array) << int64(scalar)
bool(scalar) << int64(array)
bool(array) % int64(array)
bool(array) % int64(scalar)
bool(scalar) % int64(array)
bool(array) // int64(array)
bool(array) // int64(scalar)
bool(scalar) // int64(array)
bool(array) >> int64(array)
bool(array) >> int64(scalar)
bool(scalar) >> int64(array)
bool(array) != int64(array)
bool(array) != int64(scalar)
bool(scalar) != int64(array)
bool(array) == int64(array)
bool(array) == int64(scalar)
bool(scalar) == int64(array)
bool(array) >= int64(array)
bool(array) >= int64(scalar)
bool(scalar) >= int64(array)
bool(array) | int64(array)
bool(array) | int64(scalar)
bool(scalar) | int64(array)
bool(array) ^ int64(array)
bool(array) ^ int64(scalar)
bool(scalar) ^ int64(array)
bool(array) < int64(array)
bool(array) < int64(scalar)
bool(scalar) < int64(array)
bool(array) & uint64(array)
bool(array) & uint64(scalar)
bool(scalar) & uint64(array)
bool(array) > uint64(array)
bool(array) > uint64(scalar)
bool(scalar) > uint64(array)
bool(array) + uint64(array)
bool(array) + uint64(scalar)
bool(scalar) + uint64(array)
bool(array) <= uint64(array)
bool(array) <= uint64(scalar)
bool(scalar) <= uint64(array)
bool(array) ** uint64(array)
bool(array) ** uint64(scalar)
bool(scalar) ** uint64(array)
bool(array) / uint64(array)
bool(array) / uint64(scalar)
bool(scalar) / uint64(array)
bool(array) << uint64(array)
bool(array) << uint64(scalar)
bool(scalar) << uint64(array)
bool(array) % uint64(array)
bool(array) % uint64(scalar)
bool(scalar) % uint64(array)
bool(array) * uint64(array)
bool(array) * uint64(scalar)
bool(scalar) * uint64(array)
bool(array) // uint64(array)
bool(array) // uint64(scalar)
bool(scalar) // uint64(array)
bool(array) >> uint64(array)
bool(array) >> uint64(scalar)
bool(scalar) >> uint64(array)
bool(array) != uint64(array)
bool(array) != uint64(scalar)
bool(scalar) != uint64(array)
bool(array) == uint64(array)
bool(array) == uint64(scalar)
bool(scalar) == uint64(array)
bool(array) >= uint64(array)
bool(array) >= uint64(scalar)
bool(scalar) >= uint64(array)
bool(array) | uint64(array)
bool(array) | uint64(scalar)
bool(scalar) | uint64(array)
bool(array) ^ uint64(array)
bool(array) ^ uint64(scalar)
bool(scalar) ^ uint64(array)
bool(array) - uint64(array)
bool(array) - uint64(scalar)
bool(scalar) - uint64(array)
bool(array) < uint64(array)
bool(array) < uint64(scalar)
bool(scalar) < uint64(array)
bool(array) > float64(array)
bool(array) > float64(scalar)
bool(scalar) > float64(array)
bool(array) <= float64(array)
bool(array) <= float64(scalar)
bool(scalar) <= float64(array)
bool(array) ** float64(array)
bool(array) ** float64(scalar)
bool(scalar) ** float64(array)
bool(array) / float64(array)
bool(array) / float64(scalar)
bool(scalar) / float64(array)
bool(array) % float64(array)
bool(array) % float64(scalar)
bool(scalar) % float64(array)
bool(array) // float64(array)
bool(array) // float64(scalar)
bool(scalar) // float64(array)
bool(array) != float64(array)
bool(array) != float64(scalar)
bool(scalar) != float64(array)
bool(array) == float64(array)
bool(array) == float64(scalar)
bool(scalar) == float64(array)
bool(array) >= float64(array)
bool(array) >= float64(scalar)
bool(scalar) >= float64(array)
bool(array) < float64(array)
bool(array) < float64(scalar)
bool(scalar) < float64(array)
bool(array) > bool(array)
bool(array) > bool(scalar)
bool(scalar) > bool(array)
bool(array) + bool(array)
bool(array) + bool(scalar)
bool(scalar) + bool(array)
bool(array) <= bool(array)
bool(array) <= bool(scalar)
bool(scalar) <= bool(array)
bool(array) ** bool(array)
bool(array) ** bool(scalar)
bool(scalar) ** bool(array)
bool(array) / bool(array)
bool(array) / bool(scalar)
bool(scalar) / bool(array)
bool(array) << bool(array)
bool(array) << bool(scalar)
bool(scalar) << bool(array)
bool(array) % bool(array)
bool(array) % bool(scalar)
bool(scalar) % bool(array)
bool(array) * bool(array)
bool(array) * bool(scalar)
bool(scalar) * bool(array)
bool(array) // bool(array)
bool(array) // bool(scalar)
bool(scalar) // bool(array)
bool(array) >> bool(array)
bool(array) >> bool(scalar)
bool(scalar) >> bool(array)
bool(array) != bool(scalar)
bool(scalar) != bool(array)
bool(array) == bool(scalar)
bool(scalar) == bool(array)
bool(array) >= bool(array)
bool(array) >= bool(scalar)
bool(scalar) >= bool(array)
bool(array) < bool(array)
bool(array) < bool(scalar)
bool(scalar) < bool(array)
# ops implemented by arkouda but not numpy: 15
int64(array) & float64(array)  ->  [0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00]
int64(array) << float64(array)  ->  [0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00]
int64(array) >> float64(array)  ->  [0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00]
int64(array) | float64(array)  ->  [0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00]
int64(array) ^ float64(array)  ->  [0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00]
float64(array) & int64(array)  ->  [0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00]
float64(array) << int64(array)  ->  [0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00]
float64(array) >> int64(array)  ->  [0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00]
float64(array) | int64(array)  ->  [0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00]
float64(array) ^ int64(array)  ->  [0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00]
float64(array) & float64(array)  ->  [0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00]
float64(array) << float64(array)  ->  [0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00]
float64(array) >> float64(array)  ->  [0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00]
float64(array) | float64(array)  ->  [0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00]
float64(array) ^ float64(array)  ->  [0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00 0.000000e+00]
# ops implemented by both: 239
  Matching results:         223 / 239
  Arkouda execution errors: 0 / 239

  Dtype mismatches:         7 / 239
uint64(array) > uint64(array): bool(np) vs. uint64(ak)
uint64(array) <= uint64(array): bool(np) vs. uint64(ak)
uint64(array) / uint64(array): float64(np) vs. uint64(ak)
uint64(array) != uint64(array): bool(np) vs. uint64(ak)
uint64(array) == uint64(array): bool(np) vs. uint64(ak)
uint64(array) >= uint64(array): bool(np) vs. uint64(ak)
uint64(array) < uint64(array): bool(np) vs. uint64(ak)
  Value mismatches:         9 / 239
int64(array) % float64(array): np: [       nan 0.11111111 0.22222222 0.33333333 0.44444444 0.55555556
 0.66666667 0.77777778 0.88888889 1.        ]
ak: [0. 0. 0. 0. 0. 0. 0. 0. 0. 0.]
uint64(array) & uint64(array): np: [0 1 2 3 4 5 6 7 8 9]
ak: [0 0 0 0 0 0 0 0 0 0]
uint64(array) ** uint64(array): np: [        1         1         4        27       256      3125     46656
    823543  16777216 387420489]
ak: [0 0 0 0 0 0 0 0 0 0]
uint64(array) << uint64(array): np: [   0    2    8   24   64  160  384  896 2048 4608]
ak: [0 0 0 0 0 0 0 0 0 0]
uint64(array) * uint64(array): np: [ 0  1  4  9 16 25 36 49 64 81]
ak: [0 0 0 0 0 0 0 0 0 0]
uint64(array) // uint64(array): np: [0 1 1 1 1 1 1 1 1 1]
ak: [0 0 0 0 0 0 0 0 0 0]
uint64(array) | uint64(array): np: [0 1 2 3 4 5 6 7 8 9]
ak: [0 0 0 0 0 0 0 0 0 0]
float64(array) % int64(array): np: [       nan 0.22222222 0.44444444 0.66666667 0.88888889 1.11111111
 1.33333333 1.55555556 1.77777778 2.        ]
ak: [0. 0. 0. 0. 0. 0. 0. 0. 0. 0.]
float64(array) % float64(array): np: [nan  0.  0.  0.  0.  0.  0.  0.  0.  0.]
ak: [0. 0. 0. 0. 0. 0. 0. 0. 0. 0.]
```
</details>

___________________________

Notes:
- We may want to add a `dtype`  field into `arange` at some point, but I figured building the `uint64` array with numpy and using `ak.array` would work for now
- I had to change 
```python
return new MsgTuple("Bin op not supported", MsgType.NORMAL);
```
because `create_pdarray` was struggling trying to interpret it, so I'd like @bmcdonald3 to weigh in on if the `notImplementedError` seems appropriate

Part of #1011

### CODE REVIEW

File: src/BinOp.chpl [hunk 1]
```diff
@@ -261,6 +261,8 @@ module BinOp
       var repMsg = "created %s".format(st.attrib(rname));
       return new MsgTuple(repMsg, MsgType.NORMAL);
     }
-    return new MsgTuple("Bin op not supported", MsgType.NORMAL);
+    var errorMsg = notImplementedError(pn,l.dtype,op,r.dtype);
+    omLogger.error(getModuleName(),getRoutineName(),getLineNumber(),errorMsg);
+    return new MsgTuple(errorMsg, MsgType.ERROR);
```
Comment (bmcdonald3) [inline src/BinOp.chpl:266]: > I'd like @bmcdonald3 to weigh in on if the notImplementedError seems appropriate

I am happy with this; to be honest, the error I was throwing was created only out of laziness and should have been like this from the start, so thank you for dealing with my laziness!

**Review by bmcdonald3 — APPROVED** [review]
Looks good to me, thanks for cleaning up the error message.

**Review by glitch — APPROVED** [review]

**Comment by pierce314159** [comment]
@reuster986, I added the `'unrecognized type'` check like you suggested (and the output of `testAllOperators` looks sooooo much nicer lol). I believe this is ready for re-review

**Review by bmcdonald3 — APPROVED** [review]

File: src/OperatorMsg.chpl [hunk 2]
```diff
@@ -148,7 +148,9 @@ module OperatorMsg
             return doBinOpvv(l, r, e, op, rname, pn, st);
           }
         }
-        return new MsgTuple("Bin op not supported", MsgType.NORMAL);
+        var errorMsg = notImplementedError(pn,left.dtype,op,right.dtype);
```
Comment (mhmerrill) [inline src/OperatorMsg.chpl:151]: https://github.com/Bears-R-Us/arkouda/blob/ce3c947ef03750e62035cb326a8d75b7e32eb5fe/src/OperatorMsg.chpl#L585-L590

**Review by mhmerrill — CHANGES_REQUESTED** [review]
change from `notImplementedError` to `unrecognizedTypeError`

File: src/OperatorMsg.chpl (hunk 2 above)
Comment (hokiegeek2) [reply src/OperatorMsg.chpl]: whoops, also, the MsgType should be MsgType.ERROR

**Review by mhmerrill — APPROVED** [review]
looks good!
This appears to fix an overlooked bug which pre-existed in the code ;-)

**Review by reuster986 — APPROVED** [review]

**Review by mhmerrill — APPROVED** [review]

**Pull request merged**